### DISCUSSION TITLE
Who's hiring? (Dec 2021)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#32093] [Ideas]
# Latest post (use this)
- https://github.com/vercel/next.js/discussions/35840

# What is this?
This discussion is for companies looking for people with Next.js experience. Both permanent positions and contract work are welcome.

# Companies
Please use the following format followed by a brief description of the role:

> COMPANY_NAME | ROLE | LOCATION | Full-time | $100-150k base | LINK

Please only post if you personally are part of the hiring company—no recruiting firms or job boards. Only one post per company.

Good luck with recruiting!

## Sample Job Post

Here's a good example job post.

> Vercel | Senior Full-Stack Engineer | Remote (US, Canada, Europe, or Asia) | Full-time | $100-225k base | [Link](https://vercel.com/careers/senior-full-stack-engineer-amer)
>
> Vercel’s hiring across the board on various engineering teams and positions, but the above link is for our full-stack engineering position. Vercel is innovating on and for the web and at the edge. We build tools and workflows developers love and deliver outstanding performance for end users.
>
> I've been at the company for 2 years and still find plenty of opportunity for impact, and love the teammates I'm working with. Feel free to apply using the link above or reach out to me directly.

---

*If you're looking for last month's discussion, see #31542. At least one company has hired someone as a result of it!* 

If you're a developer looking to post about themselves, use #32094.

**Comment by cole-robertson** [comment]
Tropic | Frontend Software Engineer | Remote | Full-time | 100k+ base + equity | [Link](https://jobs.lever.co/tropic/29871cc5-4593-40e4-868f-8256788f490e)

Hey, I'm a senior software engineer at Tropic and we are hiring for a REMOTE frontend React engineer!

DM me and I can fast track the process!

You'll be working closely with our product and head of design to translate grade A Figma designs into React code. Our team is incredible and using all the latest and greatest including Storybook, Chromatic, react-query, Chakra UI, etc. 

Business has been booming and the benefits of working in this startup are just amazing. 

- Competitive salary and meaningful equity
- Yearly remote office stipend
- 75% employer match on medical, dental, and vision
- 401k program
- Generous parental leave policy
- Unlimited PTO with a minimum # of days to make sure you have some time to get away to somewhere tropical

Can't wait to hopefully work with one of you from this awesome community!

**Reply by AwkwardKore** [reply]
Hi! I'm interested in this position. I just sent my information on the link. Looking forward to hearing back from you! 😄

**Reply by HeyClos** [reply]
Hi Cole, I've just sent in my application via the link you provided. Tropic looks like a very cool company to work at!

**Reply by Jassi10000-zz** [reply]
@cole-robertson 
Here's my resume : https://drive.google.com/file/d/1ncuPp_9GAbCLj-KBl4JvgK3PnFObeomv/view?usp=sharing 
If you find me a fit , pls feel free reach me out on any of the socials mentioned in my resume

**Reply by araborg** [reply]
Hi cole-robertson. I am interested in the job. Here is my resume: https://arabambi-sam-cv.netlify.app/

**Reply by nikoescobal** [reply]
> Tropic | Frontend Software Engineer | Remote | Full-time | 100k+ base + equity | [Link](https://jobs.lever.co/tropic/29871cc5-4593-40e4-868f-8256788f490e)
> 
> Hey, I'm a senior software engineer at Tropic and we are hiring for a REMOTE frontend React engineer!
> 
> DM me and I can fast track the process!
> 
> You'll be working closely with our product and head of design to translate grade A Figma designs into React code. Our team is incredible and using all the latest and greatest including Storybook, Chromatic, react-query, Chakra UI, etc.
> 
> Business has been booming and the benefits of working in this startup are just amazing.
> 
> * Competitive salary and meaningful equity
> * Yearly remote office stipend
> * 75% employer match on medical, dental, and vision
> * 401k program
> * Generous parental leave policy
> * Unlimited PTO with a minimum # of days to make sure you have some time to get away to somewhere tropical
> 
> Can't wait to hopefully work with one of you from this awesome community!

Is this still open by any chance?

**Reply by PrimozRome** [reply]
The link to https://jobs.lever.co/tropic/29871cc5-4593-40e4-868f-8256788f490e does not work. I am interested as well. 15+ years experience in web (Front and Backend).

**Comment by whoisryosuke** [comment]
PlayStation | Sr. Design Technologist | SF, LA | Full-time | [Link](https://boards.greenhouse.io/sonyinteractiveentertainmentplaystation/jobs/3682884)

Design technology is a key pillar of design execution at PlayStation. Through exploratory prototyping and tools development, Design Technologists accelerate innovation timelines and allow us to achieve more elegant and polished final products.

I’ve been at PlayStation for a over a year now and I find I can still make a high impact, explore new technology, and have a great work-life balance. You’d be joining my team for this specific position, so feel free to reach out with questions ☑️ 💬

**Comment by tbgse** [comment]
Ace & Tate | Frontend Engineer | Europe - Amsterdam (Remote possible on case by case basis) | Full-time | [Link](https://jobs.aceandtate.com/posting/14e4bdb7-4195-4c4d-ad9d-88c24e985132?section=offices)

We're pretty heavily invested into Next + serverless architecture, so if you're into all things Jamstack, there is a lot of space to learn and experiment here. You will work closely with our UI/UX designers to bridge the gap between visual design and technical implementation. Taking an active role in defining how our storefront (web & POS) looks as well as how it works. Speed and performance are key. Some interesting topics you'd be diving into over the next year are personalization & experimentation on the edge, optimizing our SSG setup by reducing payload data and various other user facing features.

If this sounds like a good fit, drop me a message and we'll set up an informal chat.

**Reply by Jassi10000-zz** [reply]
@tbgse
Here's my resume : https://drive.google.com/file/d/1ncuPp_9GAbCLj-KBl4JvgK3PnFObeomv/view?usp=sharing 
If you find me a fit , pls feel free reach me out on any of the socials mentioned in my resume

**Comment by droegier** [comment]
Jurata | Frontend Developer | Porto / Remote | Full-time | [Link](https://jurata.join.com/jobs/3443739-react-developer)

We're an early-stage startup closing up on an investment round and are looking for someone to slowly take over the reins of our Next.js frontend. There is a lawyer search with thousands of profile pages in place, as well as an authenticated backend, built on Apollo GraphQL. Initially, we will need to update some frameworks to the latest version and do a bit of refactoring before picking up the roadmap again. Besides this, as we are a small team, there is the space to take up a wide variety of tasks and grow with the company over the coming years.

If this sounds interesting to you, drop a reply here or get in touch over the link above, so that we can get to know each other.

**Reply by HeyClos** [reply]
I'm interested! feel free to reach out at thiscarlos@gmail.com or https://www.linkedin.com/in/carlosdesantiago/

**Reply by Jassi10000-zz** [reply]
@droegier 
Here's my resume : https://drive.google.com/file/d/1ncuPp_9GAbCLj-KBl4JvgK3PnFObeomv/view?usp=sharing 
If you find me a fit , pls feel free reach me out on any of the socials mentioned in my resume

**Reply by azakero** [reply]
Hi there. I'm interested in the position. Please have a look at my resume and portfolio website below.

Resume/CV: https://drive.google.com/file/d/1oKX6L4-or6hAbVxP01qf68uio9-wRnzD/view
Website: https://www.zaker.dev
Email: gloryfaircompany@gmail.com

**Reply by erinoggz** [reply]
Hi, I am interested in this position.
https://drive.google.com/file/d/1oXKfEa6QHfCd2hrc18udSsgRf4046FOI/view?usp=drivesdk
Please check my resume and let me know your decision.

Thank you

**Comment by rafaelderolez** [comment]
**Timeline Nutrition | Frontend Engineer | Europe / Remote | 80-100% for Q1 2022 with possible extension | [Link](https://www.timelinenutrition.com/)**

You’d be joining our existing team of designers, developers and marketing experts,
to help us deliver a headless e-commerce solution based on Next.js, Shopify, and Sanity.

Interested in helping me build the e-commerce side of the site? Reply down below or get in touch via email (on my profile).

_[PDF version](https://github.com/vercel/next.js/files/7668672/Front-End.Developer.Job.Offer.pdf)_

<details>
  <summary>Tech stack</summary>

- Next.js deployed to Vercel
- Sanity.io for content management
- GraphQL and REST for the integration with various back-end systems
- Shopify and Magento (legacy) for the e-commerce back-end
- Salesforce for CRM
</details>

<details>
  <summary>Wish list</summary>

- 3+ years of experience with Javascript web applications in a professional setting (we’d love
to see your portfolio)
- Proficient with at least one high-level JS framework: React, Angular, Vue or other
- Experience with the JamStack approach is a plus (i.e. Next.js, Gatsby)
- CI/CD approach to development (Git, build pipelines, packaging, deployment)
- Headless e-commerce experience is a big plus
- Experience building APIs is a plus
- Understanding of SEO principles is a plus
- Serverless (AWS Lambda or similar) experience is a plus
</details>

**Reply by Jassi10000-zz** [reply]
@rafaelderolez 
Here's my resume : https://drive.google.com/file/d/1ncuPp_9GAbCLj-KBl4JvgK3PnFObeomv/view?usp=sharing 
If you find me a fit , pls feel free reach me out on any of the socials mentioned in my resume

**Reply by azakero** [reply]
Hi there. I'm interested in the position. Please have a look at my resume and portfolio website below.

Resume/CV: https://drive.google.com/file/d/1oKX6L4-or6hAbVxP01qf68uio9-wRnzD/view
Website: https://www.zaker.dev
Email: gloryfaircompany@gmail.com

**Comment by ardiewen** [comment]
#### Hunterlily & Co. | Freelance Developer(s) | Toronto, ON or Remote (Canada) | Freelance / Contract | $85-125/hr CAD

Hi all, I run a (currently local) network of freelancers who collaborate on various first-party headless CMS and headless eCommerce projects, as well as third-party subcontracting (e.g. resource supplementation) to other agencies.

We operate exclusively on a network of freelance talent, including strategists, designers, PMs, and developers. We're looking to add more development talent into our network for 2022.

Our projects generally run a minimum of 80 billable hours on the developer side, often with additional phases of work unlocked as we go.

Communications are done mostly async and work flexibility is extremely high (20-25hrs a week is generally enough for most projects).

### What we're looking for

- **Canadian / able to work anywhere in Canada** (preferably with HST registration). This is non-negotiable unfortunately.
- Experience with Gatsby / NextJS and open to tinkering with Hydrogen (Shopify's new framework)
- Experience working with visual designers, and implementing pixel-perfect representations of design
- Experience with graphql, specifically in Gatsby and/or Apollo Client implementations
- Experience with styled-components / emotion, and the css-in-js methodology of styling
- Experience with Wordpress (wp-graphql), and Shopify (graphql api) specifically, but knowledge of other backends welcome (Prismic, Dato, Sanity, Contentful, Strapi, etc)

Interested? Send me a note at [network@hunterlily.co](mailto:network@hunterlily.co?subject=RE:%20Hunterlily%20Freelance%20Network) and tell me a little bit about yourself.

**NB:** Please do not email if you are not Canadian / have the legal ability to work in Canada.

**Comment by everest113** [comment]
[Yottled](https://yottled.com) | Senior Software Engineer | Remoate | Full-time | **$100-150k base + meaningful equity + great benefits** | [yottled.com/careers](https://yottled.com/careers)

Yottled is the back office for service-based businesses.  What Shopify did for physical goods, Yottled is doing for services. People across the country have used Yottled to bring their business online instantly - and all by themselves.  No plugins, coding, or complex builders.

Tech you'll work with: Next.js, Typescript, GraphQL, Node, Terraform

We're looking for curious and ambitious engineers to join our team :)

**Reply by HeyClos** [reply]
just sent in my resume!

**Comment by fabien-huet** [comment]
[Chantelle](https://chantelle.com/) | senior + mid front end developers | full remote | 55-70k (see details in the package) | Europe

Chantelle is the only lingerie company that managed to keep design and manufacturing in France.

### The context

We have old e-commerce websites running on Magento and we switch them to a headless e-commerce platform (centra or bigcommerce, TBD). We need to build the front ends for those websites. The project has not completely started yet, so it would be pretty much from scratch. We need to add a senior and a mid developer to the team.

### Tech / skills that you will use

- TypeScript and NextJS/React
- GraphQL (and some REST for third party providers)
- styled components, HTML / CSS
- English (you must have good written and verbal communication skills)

### Of course

Code quality best practices, accessibility, security, performances, testing, SEO... general dev skills.
Kindness, cheerfulness, work ethic, honesty... general human skills.

### Our recruitment process:

- Send your resume / github / linkedin to fabien.huet@groupechantelle.com => screening
- Technical Interview in pair programming ~ 1 hour
- Cultural Fit Interview ~ 1 hour
- Offer presentation / Contract ~ 1 hour

If you're outside of the EU we can sponsor your VISA and offer a relocation package.

### Package

- Full remote possible, but you can come to our WeWork office in Paris and Berlin (or any WeWork office in the world)
- Flexible work hours outside of our core hours (10 to 16 CET)
- A dedicated yearly budget for your training and conferences with up to 10 dedicated days
- 25 paid leaves with extra holidays on the 24th and 31st of December
- Quarterly team retreats
- Evolution mid > senior > lead over the years.

Since we offer German or French permanent contracts the growth salaries differ, but end with the same employer cost and employee net salaries.
- for the mid (degree / training + min 2-5 years in the industry) : 55k brutto in Germany, 48k brut in France
- for the senior (degree / training + min 5-8 years in the industry) : 70k brutto in germany, 60k brut in France

**Reply by araborg** [reply]
Hi fabien-huet. I am interested in the job. Here is my resume: https://arabambi-sam-cv.netlify.app/

**Reply by azakero** [reply]
Hi there. I'm interested in the position. Please have a look at my resume and portfolio website below.

Resume/CV: https://drive.google.com/file/d/1oKX6L4-or6hAbVxP01qf68uio9-wRnzD/view
Website: https://www.zaker.dev

**Comment by ashtonlance** [comment]
Simple Focus | Front-end Developer (Mid/Senior) | REMOTE | Full-time | $80k-100k base | [LINK](https://simplefocus.com/careers/front-end-developer)

   You must have experience bringing a teammate’s designs to life, meaning you can take a flat design comp and transform it with engaging, thoughtful interactions. But don’t stop there. We want you taking strong concepts and elevate them into jaw-dropping interactive experiences by suggesting weirdo stuff that only a great developer knows how to do.

Your main focus will be on the front-end for web applications, usually with React, but we rely on all of our developers to understand and operate on the back-end as well, from building custom plugins to configuring deployment pipelines.

**Reply by thomaslnx** [reply]
Are you open to considering a junior with this stack for this position?

**Reply by ashtonlance** [reply]
@thomaslnx We're currently hiring for the mid/senior level but will be looking for some more junior devs to join our team after the new year. Go ahead and submit your info via the job posting and we'll keep your info on file.

**Reply by rbourdon** [reply]
I sent you guys an email!

**Reply by araborg** [reply]
Hi ashtonlance. I am interested in the job. Here is my resume: https://arabambi-sam-cv.netlify.app/

**Reply by azakero** [reply]
Hi there. I'm interested in the position. Please have a look at my resume and portfolio website below.

Resume/CV: https://drive.google.com/file/d/1oKX6L4-or6hAbVxP01qf68uio9-wRnzD/view
Website: https://www.zaker.dev

**Comment by crypt096** [comment]
[HIRING] [100% REMOTE] [SEVERAL POSITIONS] [FULL-TIME] 

Hey guys, I am looking for following roles: 

- DevOps/ Cloud Engineer - with Azure or AWS - mid and senior level
- Android Developers - mid and senior level 
- iOS Developers - mid and senior level 
- .NET Developers - mid and senior level 
- JavaScript with with Vue / Angular / React / React Native or Node - mid and senior level 
- Automation QA - mid and senior level

**VERY IMPORTANT: Able to work in GMT+0 - GMT+2 timezones and has +2.5yrs of experience **

If you find yourself compatible for any of those positions feel free to contact me in DM **( I will need your resume in PDF format, email and your phone number)**.
Jobs are 100% remote with very good compensation.

**Reply by araborg** [reply]
Hi crypt096.  I am interested in the job. Here is my resume: https://arabambi-sam-cv.netlify.app/

**Reply by azakero** [reply]
Hi there. I'm interested in the position. Please have a look at my resume and portfolio website below.

Resume/CV: https://drive.google.com/file/d/1oKX6L4-or6hAbVxP01qf68uio9-wRnzD/view
Website: https://www.zaker.dev

**Reply by gaurishhs** [reply]
Hi, Im interested in the job, 

Website: https://apidev234.wtf

**Comment by tkanes** [comment]
RadiLens | Full Stack Developer | Atlanta or Remote (USA) | Full-time | [Link](https://radilens.breezy.hr/p/1a6f096db151)

We’re a growing SaaS healthcare AI startup looking for full stack and Next.js experience to join us.

At [RadiLens](https://radilens.com/), we're on a mission to make sure early detection and preventative services don’t fall through the cracks in our healthcare system. We're a venture-backed, remote-first company with a hub in Atlanta (USA).

Excited to hear from you if you think there's a fit.

**UPDATE: Position filled -- thanks for those who applied!**

**Reply by Jassi10000-zz** [reply]
@tkanes 
Here's my resume : https://drive.google.com/file/d/1ncuPp_9GAbCLj-KBl4JvgK3PnFObeomv/view?usp=sharing 
If you find me a fit , pls feel free reach me out on any of the socials mentioned in my resume

**Comment by jordie23** [comment]
CarExpert.com.au | Full-stack Developer | Remote (Australia only), or Sydney, Melbourne or Brisbane | Full-time | AUD$110-140k base | [Link](https://www.seek.com.au/job/55367879)

[CarExpert](https://www.carexpert.com.au/) is a fast growing startup and we're looking for more developers to help us keep development up with our ideas. Our mission is to create the world's most trusted and enjoyable journey to new car ownership, so we are working on innovative solutions that will make researching new cars a fun and stress-free experience.

I'm the CTO. We have a team consisting of myself, 3 full-time devs & 2 contractors. We're looking for another 2 full-time full-stack developers. Daily work will vary between NextJS and our AWS backend. We're currently implementing new parts of our site using KeystoneJS CMS. 

At the moment all our developers are working remotely in various locations in Australia. You're welcome to WFH full time, or come into one of our offices if you live in Syd, Melb or Bris. You must have the right to work in Australia. Please use the seek link above for more info and to apply.

**Comment by MailysRbt** [comment]
Trendful | Software Engineer | Remote (US only) | Contractor | [Link](https://www.trendful.com/jobs/software-engineering-contractor-engcon001)

Hi all, I'm a founder @ Trendful. We are a young start-up and recently got accepted into TinySeed, an accelerator program dedicated to B2B SaaS. We are building apps to empower merchants to acquire pre-owned inventory directly from their customers.

We're looking for a full-stack dev to join us as a contractor and help us push features! We use Tailwind, Typescript, Next.js, Express.js, and Firebase.

Please use the link above to apply.

**Reply by saso124** [reply]
Hello, I am a full-stack developer. I am so interested in your job post. I hope we will discuss more detail. Thank you. I sent you friend request. I hope you accept me!

**Comment by duosana** [comment]
**Sana Benefits | Senior Fullstack Engineer | Remote (US Timezone) | Full-time | $120K - 170K+ base + equity | [Link](https://jobs.lever.co/sanabenefits/943da474-5feb-4e67-bb3d-2a9be0f6b127?lever-via=1BLCQyk8VY)**

I am an Engineering Manager at Sana Benefits and we are looking to onboard several new mid-senior level full-stack developers. We are a health insurance company that helps small businesses and medium sized companies. You can read more about us here: https://www.builtinaustin.com/2020/09/10/sana-benefits-raises-20m-series-a

We have a wonderful community of developers and a culture of growth and transparency where we enjoy supporting/mentoring each other and promoting from within. We offer competitive salaries and amazing benefits - we are a benefits company after all! Check out our [Glassdoor](https://www.glassdoor.com/Benefits/Sana-Benefits-US-Benefits-EI_IE1799796.0,13_IL.14,16_IN1.htm)!

This position is currently available to any location, but we prefer to have developers based in the US timezone for now (9-5 EST/PST). We have made accommodations in the past.

For any questions, please reach out to me #Dewo6376 or comment your information below

**Comment by gusfune** [comment]
**Divbrands | Full stack web developer (all levels) | Remote (EU Timezone) | Full-time | $20K-60K+ (depends on level)**

- We’re a 4-yr old swiss fully remote company in e-commerce.
- We want people working with Javascript, React and NodeJS. (Typescript and GraphQL is a plus).
- Main goals will be working in building our stores into Headless and integrate with different Shopify services.

Apply at: https://divbrands.recruitee.com/o/fullstack-web-developer

**Reply by araborg** [reply]
Hi gusfune. I already applied

**Reply by azakero** [reply]
Hi there. I'm interested in the position. Please have a look at my resume and portfolio website below.

Resume/CV: https://drive.google.com/file/d/1oKX6L4-or6hAbVxP01qf68uio9-wRnzD/view
Website: https://www.zaker.dev

**Comment by elsigh** [comment]
Vercel | Senior Full-Stack Engineer | Remote (US, Canada, Europe, or Asia) | Full-time | $100-225k base | [Link](https://vercel.com/careers/senior-full-stack-engineer-amer)

Vercel’s hiring across the board on various engineering teams and positions, but the above link is for our full-stack engineering position. Vercel is innovating on and for the web and at the edge. We build tools and workflows developers love and deliver outstanding performance for end users.

Feel free to apply using the link above.

**Comment by PrimozRome** [comment]
Freelance | Next.JS Developer | Remote | Project-based | Dicussible

Is there any skilled Next.js freelancers here to hire? Need someone who has experienced building multi-lingual Next.JS site with translated URL slugs and DatoCMS. CMS data schema already done, website structure is clear. Need someone with setting up a starter project with production ready deployment, with routing, CMS data fetching, Vercel/Github setup, caching, etc. No design or HTML lay-outing needed.

**Reply by adnanlah** [reply]
I am interested in this offer. You can reach me out on Twitter at adencodes or Discord: DINO#9827

**Reply by HeyClos** [reply]
If you're still looking for help feel free to shoot me an email at thiscarlos@gmail.com, I've built 2 similar starter projects.
-Carlos de Santiago

**Comment by maiklieweriesBECHTLE** [comment]
Bechtle AG | frontend developer | germany / remote possible | full-time | [Link](https://jobs.bechtle.com/job/Neckarsulm-Fullstack-JavaScript-Developer-%28wmd%29/732977702/)

The Bechtle web platform offers a rich B2B portfolio of IT products and services. We are a young team of out-of-the-box thinkers with a great tech stack and now looking for you/coders to help us migrate our platform to NextJs.

Besides developing with NextJs, you can contribute to a host of exciting things such as UI/UX design, headless CMS, or evaluating our Jamstack architecture. Join us as we build the future of Bechtle.

If you have any questions, please feel free to get in touch at [maik.lieweries@bechtle.com](mailto:maik.lieweries@bechtle.com), or apply directly via the link above.

We’re hiring for different engineering teams, levels and roles! Junior entries are very welcome. Please be advised that the primary language spoken across all teams is German.

We look forward to hearing from you!

**Comment by saso124** [comment]
could you please send me these emails to ***@***.*** This is
not my work email address. Thank you.
